docs/sources/panels-visualizations/configure-tooltips/index.md

```diff
@@ -0,0 +1,108 @@
+---
+aliases:
+keywords:
+  - grafana
+  - tooltips
+  - documentation
+labels:
+  products:
+    - cloud
+    - enterprise
+    - oss
+menuTitle: Configure tooltips
+title: Configure tooltips
+description: Configure tooltips for your visualizations
+weight: 75
+---
+
+# Configure tooltips
+
+When you hover your cursor over a visualization, Grafana can display tooltips that contain more information about a data point like the exact time of a result. You can customize tooltips to to control how many series they include and the order of those values. You can also copy the content from tooltips to use elsewhere. Learn more about configuring tooltips in [Tooltip options](#tooltip-options).
+
+## Supported visualizations
+
+You can configure tooltips for the following visualizations:
+
+|                                  |                                  |
+| -------------------------------- | -------------------------------- |
+| [Bar chart][bar chart]           | [Status history][status history] |
+| [Heatmap][heatmap]               | [Time series][time series]       |
+| [Pie chart][pie chart]           | [Trend][trend]                   |
+| [State timeline][state timeline] |
+
+<!--Also xy chart -->
+
+Some visualizations, for example [candlestick][] and [flame graph][], have tooltips, but they aren't configurable. These visualizations don't have a **Tooltip** section in the panel editor pane. [Geomaps][geomaps] provide you the option to have tooltips triggered upon click or hover under the **Map controls** options in the panel editor pane.
```

Comment:
@imatwawana, to tie into Jev's comment, Candlestick now has a configurable tooltip and section 😁

Reply:
Ha! Okay, I'll add it!